.travis.yml

```diff
@@ -3,6 +3,7 @@ language: ruby
 before_install:
   - gem update --system
   - gem install bundler
+  - gem install jbundler
```

Comment:
Man, Travis is driving me nuts. `jbundler` is in the `Gemfile`, so that wouldn't be necessary right? So frustrating.